llvm/utils/TableGen/DecoderEmitter.cpp

```diff
@@ -2603,7 +2755,7 @@ namespace {
       OpcMap;
   std::map<unsigned, std::vector<OperandInfo>> Operands;
   std::vector<unsigned> InstrLen;
-  bool IsVarLenInst = Target.hasVariableLengthEncodings();
+  const bool IsVarLenInst = Target.hasVariableLengthEncodings();
```

Comment:
We don't usually use `const` with non-pointers. Unrelated change anyway.

Reply:
Dropped.